backend/.env.example

```diff
@@ -0,0 +1,32 @@
+#
+# APPLICATION
+#
+NODE_ENV=development
+PORT=3001
+HOST=0.0.0.0
+FRONTEND_URL=http://localhost:3000
+SENTRY_DSN='your sentry dsn'
+#
+# AUTHENTICATION
+SECRET_KEY=secret
+
+#
+# DATABASE CONNECTION SECRECTS
+#
+MYSQL_USER=autoline_user
+MYSQL_DATABASE=autoline
+MYSQL_ROOT_PASSWORD=secret
+MYSQL_PASSWORD=secret
+
+# Use these for the Docker development
+MYSQL_HOST=autoline_mariadb
+MYSQL_PORT=3306
+
+# Use these for the local development
+#MYSQL_HOST=localhost
+#MYSQL_PORT=3307
```

Comment:
As we don't use Docker for the dev, let's uncomment the second `MYSQL` vars option